src/sections/Company/Layer5-statement/index.js

```diff
@@ -6,11 +6,9 @@ import svgBackground from "../../../assets/images/homePage-images/background.svg
 import BannerSectionWrapper from "./statement.style";
 import { StaticImage } from "gatsby-plugin-image";
 
-const imgHero = "../../../assets/images/homePage-images/Lee-Calcote-Cloud-Native-Rejekts.webp";
-// const imgHero = "./images/layer5-projects.webp";
-
-
 const BannerDefault = () => {
+  const imgHero = "../../../assets/images/homePage-images/Lee-Calcote-Cloud-Native-Rejekts.webp";
```

Comment:
The reason it was declared outside is because it is better to have all the imports together and outside the component, it just keeps the code cleaner.

Reply:
Hi @Nikhil-Ladha,

Thanks, the reason I included the file inside the component was because of the way Gatsby described [using their StaticImage component](https://www.gatsbyjs.com/docs/how-to/images-and-media/using-gatsby-plugin-image/#static-images):

>Because the image is loaded at build time, you cannot pass the filename in as a prop, or otherwise generate it outside of the component. It should either be a static string, or a local variable in the component’s scope.

However, I am not sure if I am interpreting the last sentence correctly.

Cheers,
Randy

Reply:
Ah, it's fine the way it was defined earlier. What they try to say is that you cannot generate the value at build time, it should be a static value decalred within the scope of the component. In this case, is it a static variable, just with global scope.
Side note, if it would have been incorrect then the site would have never worked till now :)